.github/workflows/continuous-integration-e2e.yaml

```diff
@@ -15,8 +15,8 @@ env:
   HANDLE_PROVIDER_PARAMS: '{"baseUrl":"http://localhost:4011/"}'
   STAKE_POOL_CONNECTION_STRING: 'postgresql://postgres:doNoUseThisSecret!@localhost:5435/stake_pool'
   STAKE_POOL_TEST_CONNECTION_STRING: 'postgresql://postgres:doNoUseThisSecret!@localhost:5435/stake_pool_test'
-  NETWORK_INFO_PROVIDER: 'http'
-  NETWORK_INFO_PROVIDER_PARAMS: '{"baseUrl":"http://localhost:4000/"}'
+  NETWORK_INFO_PROVIDER: 'ws'
```

Comment:
Since this is a PoC, is it OK to change defaults to use it?

Reply:
That's the default in development environment, this has no impact on deployed environments.

Reply:
That may be the case @iccicci but the dev or test env defaults should be aligned with the existing API

Reply:
We can revert this change before merging to `master`, but as long as this PR is open, this is a mandatory change in this branch to test the changes it contain...